### DISCUSSION TITLE
When are you going to issue 0.9.3 version?

### DISCUSSION THREAD

**Discussion by Patryx** [NRules/NRules#292] [General]
Hi,
I want to ask@ snikolayev if you are going to issue 0.9.3 version this year? Any plans?

**Comment by snikolayev** [comment]
@Patryx great timing. I'm literally about to cut the release

**Comment by snikolayev** [comment]
And it's out: https://github.com/NRules/NRules/releases/v0.9.3

**Reply by Patryx** [reply]
Thanks!
